src/Uno.UWP/Storage/StorageFile.cs

```diff
@@ -46,6 +46,15 @@ public async Task DeleteAsync(CancellationToken ct)
 			fileInfo.Delete();
 		}
 
+		public global::System.DateTimeOffset DateCreated
```

Comment:
Use lambda expression body (e.g. `DateCreated => File.GetCreationTime(Path);` )

Reply:
I'm not good in lambda expression... How compiler knows that I want only get{}, not set{}?

Reply:
Please [see this](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members#read-only-properties)